highfive/newpr.py

```diff
@@ -269,12 +284,16 @@ def choose_reviewer(repo, owner, diff, exclude, config):
     if exclude in reviewers:
         reviewers.remove(exclude)
 
+    mention_list = None
     if reviewers:
         random.seed()
-        return random.choice(reviewers)
-    else:
-        # no eligible reviewer found
-        return None
+        if len(to_mention) > 0:
```

Comment:
Both options are implicit and you still have to test for both cases where `to_mention` is used, so I don't think there is any reason to do this. The only reason to do this would be if 'no list' has different semantics from 'empty list', and here they do not.